### PR TITLE
Multiple swagger api project

### LINKED ISSUE #204: Multiple swagger file support with one API CRD
**Description:**
Deploy multiple swagger files (multiple API Projects) as one micro-gateway image by creating one API CRD.

- API CRD should supports multiple swagger-configmaps.

**Related Issues:**
https://github.com/wso2/K8s-api-operator/issues/224

### PR DESCRIPTION
**Pull request by renuka-fernando**
fix #204 

#### Description:
Deploy multiple swagger files (multiple API Projects) as one micro-gateway image by creating one API CRD.

- API CRD should supports multiple swagger-configmaps.

```sh
apictl add api -n pets \
-f scenarios/scenario-6/swagger.yaml \
-f scenarios/scenario-10/petstore-int \
-f scenarios/scenario-3/swagger.yaml \
--mode privateJet \
--version v2.1.2
```

- will refactor `apictl update` command in another PR

### CODE REVIEW

**Comment by Rajith90** [comment]
Can I put all the swaggers into a folder and give that folder as the input

**Comment by renuka-fernando** [comment]
> Can I put all the swaggers into a folder and give that folder as the input

no.
should give each file and each API project
`-f swagger-1.yaml -f swagger-2.yaml... -f api-project-1 -f api-project-2...`

**Comment by Rajith90** [comment]
> > Can I put all the swaggers into a folder and give that folder as the input
> 
> no.
> should give each file and each API project
> `-f swagger-1.yaml -f swagger-2.yaml... -f api-project-1 -f api-project-2...`

hmm, ok since these projects are created using apictl , then this should be fine

File: import-export-cli/cmd/addApi.go [hunk 1]
```diff
@@ -191,70 +226,67 @@ func createAPI(name string, namespace string, configMapName string, replicas int
...
 	for _, filePath := range interceptors {
 		if filepath.Ext(filePath) == ".jar" {
 			//creating kubernetes configmap for each java interceptor
-			javaInterceptorsConfNames = append(javaInterceptorsConfNames, flagApiName + "-" + filepath.Base(filePath))
-			fmt.Println("creating configmap with java interceptor " + flagApiName + "-" + filepath.Base(filePath))
-			errConfInt := createConfigMapWithNamespace(flagApiName + "-" + filepath.Base(filePath), filePath, namespace, operation)
+			cmName := cmPrefixName + "-" + filepath.Base(filePath)
+			javaInterceptorsConfNames = append(javaInterceptorsConfNames, cmName)
+			fmt.Println("creating configmap with java interceptor " + cmName)
+			errConfInt := createConfigMapWithNamespace(cmPrefixName+"-"+filepath.Base(filePath), filePath, namespace, operation)
```
Comment (Meruja) [inline import-export-cli/cmd/addApi.go:273]: Formatting issue.
**cmPrefixName+"-"+filepath.Base(filePath)** should be changed to **cmPrefixName + "-" +  filepath.Base(filePath)**

File: import-export-cli/cmd/addApi.go [hunk 2]
```diff
@@ -191,70 +226,67 @@ func createAPI(name string, namespace string, configMapName string, replicas int
...
 			if errConfInt != nil {
-				utils.HandleErrorAndExit("Error creating configmap for java-interceptor" + flagApiName + "-" + filepath.Base(filePath), err)
+				utils.HandleErrorAndExit("Error creating configmap for java-interceptor "+cmName, errConfInt)
```
Comment (Meruja) [inline import-export-cli/cmd/addApi.go:275]: Formatting issue.

**Pull request merged**